### ISSUE TITLE
Скупщики дополнений встраивающие сбор статистики

### ISSUE THREAD

**Issue by firefoxForUser** [The-OP/Fox#40]
Как думаете стоит вам их описать?

https://addons.mozilla.org/ru/firefox/user/ppclick/
https://addons.mozilla.org/ru/firefox/addon/browserprotect/
https://addons.mozilla.org/en-US/firefox/user/6083231/

И по заявкам пополнять список.

**Comment by The-OP** [comment]
Насколько понимаю, если аддон что-то собирает, то для прохождения ревью AMO об этом должно быть упомнятуно в privacy policy аддона. Во всех остальных случаях следует писать абузу администрации AMO, чтобы убрали и вснеси в черный список.

**Comment by firefoxForUser** [comment]
Такой список интересен хотя бы ради любопытства и прослеживания тенденции. Ведь изначально некоторые из них были действительно полезными и интересными, можно сказать даже уникальными.
И ещё. Я нигде не нашел упоминания выдаётся ли какое-либо предупреждения после смены автора и политики приватности.
Вот тут https://addons.mozilla.org/ru/firefox/addon/browserprotect/versions/
видно смена лицензии и автора и сразу появился сбор статистики. Пользователям которые установили его до этого что-то показывается?

**Comment by The-OP** [comment]
>Такой список интересен хотя бы ради любопытства и прослеживания тенденции.

Я не понимаю, что ты предлагаешь. Пытаться отслеживать изменения авторов и лицензий на всем AMO? И сколько для этого нужно человеко-часов? Если хочется создать какой-то пополняемый юзерами черный список, нужен проект с намного большим числом как юзеров, так и участников.

>видна смена лицензии и автора и сразу появился сбор статистики

Где? Я там только смену лицензии вижу.

>Пользователям которые установили его до этого что-то показывается?

Рассуждая логически - вряд ли. Хотя можно проверить, поставив старую версию. Или может в [этот](https://addons.mozilla.org/firefox/addon/addon-update-checker/) аддон можно такую фичу попробовать прикрутить.

**Comment by firefoxForUser** [comment]
Не всех, а только популярных или тех на кого кто-нибудь даст ссылки (чтобы самому не искать).

Сбор статистики в browserprotect есть (хотя вижу, что там в последней версии убрали какой-то "facebook sharing", может оно). То есть я его сам проверял, устанавливал и смотрел запросы. Вот в том что на старой лицензии статистика не отправлялась, а как только лицензия поменялась - начала.

**Comment by The-OP** [comment]
Пожалуй, ты прав. Сегодня вот еще в треде запостили: https://2ch.hk/s/res/1436450.html#1437130

**Comment by firefoxForUser** [comment]
Вообще-то я тоже такого не ожидал. :) Как правило такое случается с ме проработанными дополнениями и их аторами.
На русском форуме он известен как pag77 https://forum.mozilla-russia.org/profile.php?id=67960
общается с пользователями весьма мило, в прямом смысле.

Кроме того, стал известен, как тоже немного параноик https://forum.mozilla-russia.org/viewtopic.php?pid=694631

**Comment by firefoxForUser** [comment]
Вот обсуждение на mozillazine.org http://forums.mozillazine.org/viewtopic.php?p=14377313#p14377313

**Comment by firefoxForUser** [comment]
Причем в этом дополнении реклам отключена по умолчанию и её нужно включить. Что интересно по вашей ссылке об этом не слова. :) И это действительно один из самых легких случаев.

**Comment by firefoxForUser** [comment]
Подоплека не совсем понятна. То ли разработчиков Mozilla возмутила именно целевая реклама (рекламщикам отправлялась целая страница, анализировалась и показывалась целевая реклама), то ли возмутил сам факт рекламы.

Если кто владеет английским хорошо. Постите на reddit больше инфы о рекламе и переброске/открытии дополнениями своих домашних страниц! Похоже есть шанс быть услышанными.

**Comment by The-OP** [comment]
Так, у меня наконец нашлось время для создания этого "серого списка", но что-то по ссылкам выше уже ничего нет. С чего можно начать этот список?

**Comment by nyanon** [comment]
Ого. =)  BrowserProtect столько лет спокойно себе висел там, что удаления никак не ожидалось.

Начать с профиля https://addons.mozilla.org/ru/firefox/user/ppclick/ это профиль некой компании которая суёт рекламу.
https://addons.mozilla.org/en-US/firefox/user/6083231/ - тоже самой.

Потом рассказать, что кнопочка "Политика приватности" не просто так рядом с кнопочкай "Установить". И её всегда нужно читать, правда она часто не на русском языке. =)

Потом упомянуть, что уж совсем гнусных случаев Mozilla старается не допускать. Но вот например с https://addons.mozilla.org/en-us/firefox/addon/s3download-statusbar/ случился казус. По умолчанию руклама отключена. Её можно включить если хочется автора поддержать. Вот только рекламодатель "забыл", что статистику автор https://addons.mozilla.org/en-us/firefox/user/gronar/ этого дополнения не собирался собирать. И "случайно" стал её собирать.
Так что никаких включений рекламы делать не стоит, даже если в неё написано, что она ничего не собирает.
К слову видимо из-за несоответствия заявления "статистику не собираю" с реальностью его и временно забанили в Mozilla. Плохо только что сама Mozilla этого не поняла, а панику подняло сообщество.

Думаю стоит расширить список дополнений с рекламой ещё и просто весьма странными дополнениями. Особенно теми у которых высокая популярность. Например Ghostery. Иметь проприетарную лицензию и позицианировать себя, как борца за приватность - странно.
Ещё странно, что авторы этого дополнения не могут не знать о uBlock Origin и AdBlock Plus. И знаю, что они более совершенны. Вопрос: "зачем что-то делать если знаешь что оно будет использовано толко по неграмотности?"
А ещё Ghostery также имеет отключенную по умолчанию рекламу.

Упомянуть нужно, что дополнения с желтыми кнопочками прошли только автоматизированную проверку. А как показал один хороший человек её легко обойти https://www.opennet.ru/opennews/art.shtml?num=43398
Так что устанавливать можно только дополнения с зелёными кнопочками.

**Comment by nyanon** [comment]
Вообще это будет скорее статья "Что устанавливать можно, а что нельзя"

В ней описать, что некоторые дополнения, например HTTPS Everywhere, uBlock Origin, Tree Style Tab и возможно несколько других устанавливать можно с малым риском. Потому что их авторы и сообщество вокруг них имеют репутацию.

**Comment by The-OP** [comment]
Начнем, пожалуй: 5d5f6b4

@nyanon 
1) Желтые кнопки (Preliminary Review) - это не автоматическая проверка, ревью там есть. Но от полной все же отличается.
2) Я не в курсе, что там сейчас с S3-аддонами. Поправили уже? Стоит добавлять?
3) Замечания, предложения?

**Comment by stepshal** [comment]
@The-OP http-useragent-cleaner - аддон проприетарный, укажи это.

**Comment by The-OP** [comment]
@stepshal А он точно проприетарный? У меня такое ощущение, что автор просто забыл указать лицензию.
